index.html

```diff
@@ -2039,7 +2051,7 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
                 <td>
                     0..*
                 </td>
-                <td>Point to a software implementation of the TD.</td>
+                <td>Point to the web app manifest of a web application which provides a user interface with which a user can interact with the Thing (also see [[APPMANIFEST]]).</td>
```

Comment:
This is a significant modification of the previous text.

These changes introduce a specific format to describe Web applications, which restricts the UI components to WebApps. Is this intended? 

The referenced document is a working draft, seems not to be suitable for a normative reference.

These late changes are difficult to review, and the implications cannot be properly considered in the limited review time.

Reply:
Please see #1324 

![image](https://user-images.githubusercontent.com/13832739/146944414-76a7818f-5d89-470a-a9a3-54faf0ea5d1f.png)

Reply:
I reverted this as well and put this in a separate PR #1340